src/org/parosproxy/paros/extension/report/ReportLastScan.java

```diff
@@ -180,6 +185,10 @@ public String getDescription() {
         		fileExtension=XML_FILE_EXTENSION;
         		reportXSL = null;	// Dont use XSLT
         		break;
+        	case JSON:
+                fileExtension=JSON_FILE_EXTENSION;
```

Comment:
I can see that indentation isn't consistent between XML and MD, to make the JSON case "contain" the relevant lines I'd suggest making it similar to the XML case.